scalafix-core/shared/src/main/scala/scalafix/util/TokenList.scala

```diff
@@ -8,40 +8,29 @@ import scala.meta.tokens.Tokens
 /** Helper to traverse tokens as a doubly linked list.  */
 final class TokenList private (tokens: Tokens) {
   def trailing(token: Token): SeqView[Token, IndexedSeq[Token]] =
-    tokens.view(tok2idx(token), tokens.length).drop(1)
+    tokens.view(tok2idx(token) + 1, tokens.length)
+
   def leading(token: Token): SeqView[Token, IndexedSeq[Token]] =
-    tokens.view(0, tok2idx(token)).drop(1).reverse
+    tokens.view(0, tok2idx(token)).reverse
```

Comment:
I believe the goal here was to have the `token` element dropped from the result. However, this is already handled by the `.view` method as the `end` parameter is exclusive

Reply:
Woah, this was surprising :D 

```scala
@     tokens.view(0, tok2idx(token)).reverse.mkString
res19: String = "(list { a object"

@     tokens.view(0, tok2idx(token)).drop(1).reverse.mkString
res20: String = "(list { a object"
```

Yes, the objective was to drop the token.

Reply:
```scala
val tokens = "foo bar".tokenize.get
tokens: scala.meta.tokens.Tokens = Tokens(, foo,  , bar, )
val barIdx = 3

tokens.view(0, barIdx).reverse.mkString("|")
res11: String = " |foo|"
tokens.view(0, barIdx).drop(1).reverse.mkString("|")
res12: String = " |foo"
```

Reply:
I see, `drop(1)` was removing the `BOF` token, which is wrong behaviour but probably never surfaced as a problem because it's zero length. Good catch!